perception/image_projection_based_fusion/include/image_projection_based_fusion/fusion_node.hpp

```diff
@@ -76,30 +83,40 @@ class FusionNode : public rclcpp::Node
 
   void publish(const Msg & output_msg);
 
+  void timer_callback();
+  void setPeriod(const int64_t new_period);
+
   std::size_t rois_number_{1};
   tf2_ros::Buffer tf_buffer_;
   tf2_ros::TransformListener tf_listener_;
 
   // camera_info
   std::map<std::size_t, sensor_msgs::msg::CameraInfo> camera_info_map_;
   std::vector<rclcpp::Subscription<sensor_msgs::msg::CameraInfo>::SharedPtr> camera_info_subs_;
+  std::vector<double> tan_h_;  // horizontal field of view
```

Comment:
I think this variable is better to be moved to `pointpainting/fusion.hpp` since it's used only for pointpainting.

Reply:
solved in https://github.com/autowarefoundation/autoware.universe/pull/2175/commits/7807a448f3c772d49ebe50bc6f5fe3863ead6a64